server/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/GrammarPreprocessorListenerImpl.java

```diff
@@ -276,6 +287,29 @@ public void exitIncludeStatement(@NonNull IncludeStatementContext ctx) {
         MAX_COPYBOOK_NAME_LENGTH_DATASET);
   }
 
+  private void skipCopybook(
+      ParserRuleContext context,
+      CopySourceContext copySource,
+      Locality copybookStatementPosition,
+      int maxLength) {
+
+    String copybookName = retrieveCopybookName(copySource);
+    Locality locality = retrievePosition(copySource);
+    CopybookModel model = getCopyBookContent(copybookName, locality, copybookConfig);
```

Comment:
If we ignore the copybook does it make sense to ignore it completely? Because here we will link it with the file we use in other divisions and also add it into downloading queue.

Reply:
Oops, I missed that call, thank you. 
I decided to save the behavior of the copybook except adding its content to the document just to keep the consistent behavior.